tests/lorem-ipsums.py

```diff
@@ -21,25 +37,67 @@ def remove_comments(text_string):
     return p
 
 
-def main():
-    """Main function
-	Checks for rogue "lorem ipsum"s in book content files and removes html
-	comment identifiers from the files.
+def check_changed_files(pr_num, bad_phrase=BAD_PHRASE):
+    """Check the files in a Pull Request for an undesirable phrase
 
-	Raises:
-		Exception: Raises an exception if "lorem ipsum" is found in a book
-					content file
-	"""
+    Arguments:
+        pr_num {str} -- Number of the Pull Request with modified files
+
+    Keyword Arguments:
+        bad_phrase {str} -- The undesirable phrase to check for (default: {BAD_PHRASE})
+
+    Returns:
+        {list} -- List of filenames that contain the undesirable phrase
+    """
+    filenames = filter_files(pr_num)
+    failed = []
+
+    for filename in filenames:
+        f = open(
+            os.path.join(ABSOLUTE_HERE, filename), encoding="utf8", errors="ignore"
+        )
+        text = f.read()
+        text = remove_comments(text)
+        if bad_phrase in text.lower():
+            failed.append(filename.name)
+
+    return failed
+
+
+def check_all_files(
+    bad_phrase=BAD_PHRASE,
+    directory_to_check=os.path.join(ABSOLUTE_HERE, "book", "content"),
```

Comment:
I think this path might not be found and trigger an error with the new build of the book

Reply:
Good catch on these, I've updated both!